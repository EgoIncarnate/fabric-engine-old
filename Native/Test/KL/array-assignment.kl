operator entry()
{
  var Integer a[];
  a.push( 17 );
  report a;

  var Integer b[];
  b = a;
  report b;
}
