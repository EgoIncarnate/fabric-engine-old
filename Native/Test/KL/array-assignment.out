GlobalList
  Function( 'entry' ('entry'), void )
    ParamList
    CompoundStatement
      StatementList
        StatementList
          VarDecl( 'a', Integer[] )
        ExprStatement
          MethodOp( 'push' )
            Var('a')
            ArgList
              ConstSize( 17 )
        Report
          Var('a')
        StatementList
          VarDecl( 'b', Integer[] )
        ExprStatement
          AssignOp(ASSIGN_OP)
            Var('b')
            Var('a')
        Report
          Var('b')
[17]
[17]
